### PR TITLE
Add rspec matcher descriptions rebase on master

### PR DESCRIPTION
**Pull request by presidentJFK**

### CODE REVIEW

**Comment by presidentJFK** [comment]
@arronmabrey I rebased your branch on master, after merging some spec fixes into master. I want to see if CI is all passing while I review, which I am doing now.

**Comment by arronmabrey** [comment]
Sounds good. Let me know if there are any issues. :-)

File: lib/webmock/request_execution_verifier.rb [hunk 1]
```diff
@@ -67,6 +51,25 @@ def self.executed_requests_message
 
     private
 
+    def failure_message_phrase(is_negated=false)
```
Comment (presidentJFK) [inline lib/webmock/request_execution_verifier.rb]: :+1: nice refactor

Comment (arronmabrey) [inline lib/webmock/request_execution_verifier.rb]: Thanks :-)

**Pull request closed without merging**